contracts/interfaces/IUniAgent.sol

```diff
@@ -4,18 +4,29 @@ pragma solidity >=0.8.0;
 /// @title IUniAgent Interface
 /// @author imToken Labs
 interface IUniAgent {
-    error ZeroAddress();
     error InvalidMsgValue();
     error UnknownRouterType();
```

Comment:
I think we don't need this error because solidity automatically checks the validity of enum input for us (it would just raise EVM error. So it would never reach the code path here in `_getRouterAddress`:
```solidity
    function _getRouterAddress(...) ... {
        ...
        } else {
            revert UnknownRouterType();
        }
    }
```

Reply:
Removed the error but in order to avoid compiler warning (uncovered code path), revert without msg as default.

Reply:
5ae24e4